en/docs/services/create-your-first-service.md

```diff
@@ -1,168 +1,161 @@
     
         ![Select HTTP Connection](../assets/img/services/select-http-connection.png)
-        
-    2. Enter information as follows:
-    
-        | **Field**           | **Value**                    |
-        |---------------------|------------------------------|
-        | **Connection Name** | `httpEndpoint`               |
-        | **URL**             | `"https://api.covid19api.com"` |
-        
-        Click **Save**.
-        
+         
+    2. Enter the following information and then click **Save**:
+    
+        | **Field**           | **Value**                      |
+        |---------------------|--------------------------------|
+        | **Connection Name** | `httpEndpoint`                 |
+        | **URL**             | `https://api.covid19api.com`   |
+         
     3. Click the **+** icon below the HTTP API call you added. 
     
         ![Update Existing Connection](../assets/img/services/update-existing-connection.png)
         
-    4. Click **API Calls**, and then click on the existing connection.
+    4. Click **API Calls** and then click the existing connection.
     
         ![Select Existing connection](../assets/img/services/select-existing-connection.png)
         
-    5. Enter information as follows:
-        
-        ![Configure HTTP Connection](../assets/img/services/select-existing-connection.png)
-        
-        | **Field**               | **Value**                      |
-        |-------------------------|--------------------------------|
-        | **OPERATION**           | **get**                        |
-        | **Resource Path**       | `"/total/country/united-states"` |
-        | **Select Payload Type** | **JSON**                       |
+    5. Enter the following information:
         
+        | **Field**               | **Value**                       |
+        |-------------------------|---------------------------------|
+        | **OPERATION**           | **get**                         |
+        | **Resource Path**       | `"/total/country/united-states"`|
+        | **Select Payload Type** | **JSON**                        |
+
         !!! info
             To select **JSON** as the payload type, you need to click **Do you want to extract a payload?**.
             ![Do you want to extract a payload button](../assets/img/services/enable-payload.png)
-        
-        Click **Save & Done**.
-        
-        Now your API trigger is connected to the public API that fetches the active COVID-19 cases in the selected country.
+            
+    6. Click **Save & Done**.
     
-7. To cast all the data retrieved via the API connected to the service before further processing, add a new custom statement as follows:
+       Now your API trigger is connected to the public API that fetches the active COVID-19 cases in the selected country.
+
+7. Add a new custom statement as follows to cast all the data retrieved via the API connected to the service before further processing:
```

Comment:
what we are trying to do here, and why, is not very clear.

Reply:
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108